app/controllers/application_controller.rb

```diff
@@ -5,7 +5,8 @@ class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name address postal_code])
-    devise_parameter_sanitizer.permit(:account_update, keys: %i[name address postal_code self_introduction])
+    permitted_params = %i[name address postal_code]
+    devise_parameter_sanitizer.permit(:sign_up, keys: permitted_params)
+    devise_parameter_sanitizer.permit(:account_update, keys: permitted_params << :self_introduction)
```

Comment:
```suggestion
    devise_parameter_sanitizer.permit(:account_update, keys: [:self_introduction, *permitted_params])
```

ここでは`<<`を使って要素を追加するよりも、↑のようにして新しい配列を作る方が自然かなと思います。
（`<<`は要素の追加のためのメソッド（演算子）であって、Rubyの慣習的にその戻り値を直接使用することがあまりないため）

Reply:
`*`を使って新しく配列を作り直すやり方は全く思いつかなかったので目から鱗でした！